src/IonTimestamp.ts

```diff
@@ -270,13 +279,21 @@ export class Timestamp {
      *
      * Note that a return value of 0 doesn't guarantee that equals() would return true,
      * as compareTo() doesn't require the values to have the same precision to be considered equal,
-     * whereas equals() does.
+     * whereas equals() does.  For example, calling compareTo() for any of the following timestamps
+     * will produce a result of 0, but equals() will return false, as they have different precisions
+     * or local offsets:
+     *
+     *     2001T
```

Comment:
I would add the examples like so 

|Timestamp 1| Timestamp 2| `compareTo`| `equals`|
| -----------------| ------------------|------------------|-------------|
|2001T           | 2001-01-01T|  0                    |  `false`    |

Reply:
ooh, fancy markdown.  :)  i've switched to table form, and agree that this adds clarity